src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java

```diff
@@ -949,4 +976,57 @@ private void openCodebashingLink() throws CxConfig.InvalidCLIConfigException, IO
                 Utils.getLogger(ResultNode.class).error(error.getMessage(), error);
             }
     }
+
+    private void generateLearnMore(@NotNull LearnMore learnMore, JPanel panel) {
+        JLabel riskTitle = new JLabel(String.format("<html><b>%s</b></html>", boldLabel(Bundle.message(Resource.RISK)).getText()));
+        panel.add(riskTitle, "span, growx");
+
+        String risk = learnMore.getRisk();
+        if (StringUtils.isNotBlank(risk)) {
+            // wrapping the description in html tags auto wraps the text when it reaches the parent component size
+            panel.add(new JBLabel(String.format(Constants.HTML_WRAPPER_FORMAT, risk.replaceAll("\n","<br/>"))),
+                    "wrap, gapbottom 3, gapleft 0");
+        }
+
+        JLabel causeTitle = new JLabel(String.format("<html><b>%s</b></html>", boldLabel(Bundle.message(Resource.CAUSE)).getText()));
+        panel.add(causeTitle, "span, growx");
+
+        String cause = learnMore.getCause();
+        if (StringUtils.isNotBlank(cause)) {
+            // wrapping the description in html tags auto wraps the text when it reaches the parent component size
+            panel.add(new JBLabel(String.format(Constants.HTML_WRAPPER_FORMAT, cause.replaceAll("\n","<br/>"))),
+                    "wrap, gapbottom 3, gapleft 0");
+        }
+
+        JLabel recommendationsTitle = new JLabel(String.format("<html><b>%s</b></html>", boldLabel(Bundle.message(Resource.GENERAL_RECOMMENDATIONS)).getText()));
+        panel.add(recommendationsTitle, "span, growx");
+
+        String recommendations = learnMore.getGeneralRecommendations();
+        if (StringUtils.isNotBlank(cause)) {
+            // wrapping the description in html tags auto wraps the text when it reaches the parent component size
+            panel.add(new JBLabel(String.format(Constants.HTML_WRAPPER_FORMAT, recommendations.replaceAll("\n","<br/>"))),
+                    "wrap, gapbottom 3, gapleft 0");
+        }
+    }
+
+    private void generateCodeSamples(@NotNull LearnMore learnMore, JPanel panel) {
+        List<Sample> samples = learnMore.getSamples();
+        if(samples.size()>0){
+            for (Sample sample : samples) {
+                String title = sample.getTitle();
+                panel.add(new JBLabel(String.format(Constants.HTML_WRAPPER_FORMAT, title +" using "+ sample.getProgLanguage())),
```

Comment:
@tiagobcx we can also make this a constant

Reply:
Will do, thanks.